app/models/list.rb

```diff
@@ -1,7 +1,9 @@
 class List < ApplicationRecord
+  acts_as_taggable
+
   has_and_belongs_to_many :homepages
   belongs_to :user
-
+ 
```

Comment:
Yeah that extra space, not the newline.

On Wed, Aug 3, 2016, 12:46 PM wsf246 <notifications@github.com> wrote:

> In app/models/list.rb
> <https://github.com/briankung/projectcredo/pull/43#discussion_r73377383>:
>
> >    has_and_belongs_to_many :homepages
> >    belongs_to :user
> > -
> > +
>
> Or did you mean the ' ' that I just noticed is there?
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/briankung/projectcredo/pull/43/files/d5d35f536514f294ca290898c3d3e2b5932618b3#r73377383>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACtGx4MCxbtyKtQkn34gxcHDQVZWqa11ks5qcNKzgaJpZM4JbLTZ>
> .
>